Sprint-2/debug/author.js

```diff
@@ -11,6 +11,10 @@ const author = {
   alive: true,
 };
 
-for (const value of author) {
-  console.log(value);
+for (const value in author) {
```

Comment:
Good.  I think the name `value` here is a bit misleading because what the `in` operator gives you are the keys (or property names) from the object and the expression `author[value]` is actually using `value` as a **key** to return the value of that property